examples/sample_runtime/src/main.rs

```diff
@@ -265,20 +267,24 @@ fn main() {
 
     let db = TemporaryDB::new();
     let node_cfg = node_config();
-    let genesis = node_cfg.consensus.clone();
+    let consensus_config = node_cfg.consensus.clone();
     let service_keypair = node_cfg.service_keypair();
     let channel = NodeChannel::new(&node_cfg.mempool.events_pool_capacity);
     let api_sender = ApiSender::new(channel.api_requests.0.clone());
 
     println!("Creating blockchain with additional runtime...");
     // Create a blockchain with the Rust runtime and our additional runtime.
     let blockchain_base = Blockchain::new(db, service_keypair.clone(), api_sender.clone());
-    let blockchain = BlockchainBuilder::new(blockchain_base, genesis)
-        .with_rust_runtime(
-            channel.endpoints.0.clone(),
-            vec![InstanceCollection::from(SimpleSupervisor::new())],
-        )
-        .with_additional_runtime(SampleRuntime::default())
+    let supervisor_service = SimpleSupervisor::new();
+    let genesis_config = GenesisConfigBuilder::with_consensus_config(consensus_config)
```

Comment:
I suppose `GenesisConfigBuilder::new().with_consensus_config(consensus_config)` would be more idiomatic ?

Reply:
Is there any condition when we don't want to call `with_consensus_config`? I suggest to just rename this method to `new`, so it'll be `GensisConfigBuilder::new(consensus_config)`.

Reply:
However maybe it can be later.

Reply:
Well, perhaps it makes sense for [this](https://github.com/exonum/exonum/pull/1541#discussion_r352083134)

Reply:
I think that since it's not a common scenario, it's better to have a separate constructor like `without_consensus_config`.